text/dokumentation.tex

```diff
@@ -25,6 +25,9 @@
 % Folgende Zeile auskommentieren, um alle Quellenangaben im Literaturverzeichnis anzuzeigen. 
 % \newcommand{\defAlleQuellenZeigen}
 
+% Folgende Zeile auskommentieren, um die bunten Rahmen um Hyperlinks, Referenzen etc. anzuzeigen
+\newcommand{\defHideLinkBorders}
```

Comment:
du kleiner frechdachs hast das einfach umgedreht also den befehl "hide" statt "show" gemacht ;P

Reply:
würde nicht sowas funktionieren?

`\ifdefined\defShowLinkBorders pdfborder={0 0 1}, \else pdfborder={0 0 0},\fi`

also 0,0,1 ist [anscheinend ](https://tex.stackexchange.com/questions/175385/what-do-the-first-two-numbers-in-the-pdfborder-option-of-hyperref-do)der default

Reply:
Ja ach was, bin ja nich blöd ... ;)